src/lib/registration/dialogs/online_search.rb

```diff
@@ -83,7 +83,7 @@ def help
           "after a few seconds, shows the results in the table, including the module/extension " \
           "each package belongs to.</p>\n" \
           "<p>You can select any package for installation by clicking in the corresponding table " \
-          "row and pressing the <b>Select package</b> button (or just double-clicking on the " \
+          "row and pressing the <b>Toggle selection</b> button (or just double-clicking on the " \
           "row). If the package belongs to a not activated module/extension, YaST asks you about " \
           "activating it. Bear in mind that the real activation takes place after you click the " \
           "<b>Next</b> button.</p>\n")
```

Comment:
I'm just thinking whether we should use some placeholders for all button labels here, e.g.

```suggestion
          "<b>%{next}</b> button.</p>\n" % {next: Label.NextButton})
```

- This would ensure the "Next" is translated consistently
- Changing the label does not invalidate the existing translations, quite important for such a long text...

Reply:
Thanks a lot for the suggestion @lslezak 

I really like the idea, but I'd prefer to update all of them (_Search_, _Package Search_, _Toggle selection_, etc) or none. But not only for the _Next_ one, although I guess you simply put an example.

To do so, we need to change all involved widgets, to have class methods exposing all labels, but I'd keep the translation as it is now, postponing the discussion to see if we can manage to use this approach easily in future developments.

Thanks again!